workflows/nanopore.nf

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
In the future, I'd just not include incomplete sections of code. I generally see `main` as always fully operational and this basically means if someone pulls main everything should work as expected. 

If you want to do this, I'd have a feature branch that goes in parallel to `main` and you open mini-PRs from whatever develop branch to that feature branch and when the feature branch is good to go, you can do a big PR from feature branch to `main`. That big PR would most likely not be an issue because all the sub-components would be already reviewed.

Reply:
I see - this PR is more of a mini PR where the next part of more "complete" additions are in #26, this was just staging the separation of the illumina and nanopore workflows, so the `--platform` flag is necessary to distinguish between the two